### PR TITLE
Add new Project to README.md and index.md

### LINKED ISSUE #197: Project
Hi, 
besides the Twitter account (https://twitter.com/GermanyCovid) there is now also a discordbot with this API. You can invite it at [germanycovid.de/discord](https://germanycovid.de/discord). I would be happy if this bot could also be listed.

### PR DESCRIPTION
**Pull request by Rubber1Duck**
Fix #197

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -137,6 +137,7 @@ Now you can access the server at `http://localhost:8080`.
 - Corona, what's going on in your county or city [https://chrnoe.de/corona](https://chrnoe.de/corona)
 - Wordpress Widget zur Anzeige der letzen 3 Inzidenzen für einen Landkreis [https://github.com/p-tenz/widget-7d-incidence](https://github.com/p-tenz/widget-7d-incidence)
 - Informatives Dashboard, zum schnellen Überblick des Status der Corona-Schutzimpfungen in Deutschland [https://www.impfung.io/](https://www.impfung.io/)
+- - Discordbot at [https://germanycovid.de/discord](https://germanycovid.de/discord)
```
Comment (marlon360) [inline docs/index.md]: There should only be a single dash „-“

**Review by marlon360 — CHANGES_REQUESTED** [review]
Thank you! But I have to request a minor change.

File: docs/index.md (hunk 1 above)
Comment (Rubber1Duck) [reply docs/index.md]: done

**Review by marlon360 — APPROVED** [review]
Thanks!

**Pull request merged**